dev/Gemfile.lock

```diff
@@ -0,0 +1,209 @@
+GEM
```

Comment:
Is this file useful to you? Feel free to gitignore it if it's not. 

Looked like you and I have different versions of Bundler installed (which is totally fine), but that meant every time I ran bundler it would change this file with the Bundler version I was using. I haven't been working in `dev/` so not really a bother at this point, just saying.